node-core/services/registry/registry.go

```diff
@@ -151,3 +156,22 @@ func (s *Registry) FetchService(service interface{}) error {
 	}
 	return errUnknownService
 }
+
+func (s *Registry) CommitMultiStore() store.CommitMultiStore {
+	// Consensus service must expose the CommitMultistore. We
+	// retrieve it by its name and check it implements the accessor
+	svc, found := s.services[cservice.AppName]
+	if !found {
+		err := fmt.Errorf("service %s not found", cservice.AppName)
+		panic(err)
+	}
+
+	sh, ok := svc.(CommitMultistoreAccessor)
+	if !ok {
+		err := fmt.Errorf("service %s does not implement CommitMultistoreAccessor",
+			cservice.AppName,
+		)
+		panic(err)
+	}
+	return sh.CommitMultiStore()
```

Comment:
we could do this in NewRegistry, but I believe this would cause tests like `TestRegistry_FetchService` and all the testing built on top of it more complex